packages/babel-plugin-transform-classes/src/spec.js

```diff
@@ -0,0 +1,38 @@
+import VanillaTransformer from "./vanilla";
+import { types as t } from "@babel/core";
+
+export default class SpecClassTransformer extends VanillaTransformer {
```

Comment:
Rather than inheritance I would prefer composition, it's not currently an issue but could cause unexpected results in the future (from maintenance/interations).